src/06-loops.js

```diff
@@ -165,7 +240,21 @@ function timesIndex(array) {}
  *  //> [ 5, 5 + 2, 5 + 2 + 9 ]
  *  //> [ 5, 7, 16 ]
  */
-function cumulativeSum(array) {}
+function cumulativeSum(array) {
+  thatSumArray = [];
+  value = 0;
```

Comment:
Ohhh noes! We accidentally declared a global variable here by forgetting to declare it.